pkg/health/health_job.go

```diff
@@ -50,6 +58,11 @@ func getBatchv1JobHealth(job *batchv1.Job) (*HealthStatus, error) {
 			Status:  HealthStatusDegraded,
 			Message: failMsg,
 		}, nil
+	} else if isSuspended {
+		return &HealthStatus{
+			Status:  HealthStatusSuspended,
+			Message: failMsg,
```

Comment:
@ashutosh16 @alexmt Why does this return `failMsg`, which might not be set at all in the Suspended case?